### ISSUE TITLE
tray dissappears after restarting waybar "no such object path /StatusNotifierWatcher"

### ISSUE THREAD

**Issue by Shmerlard** [Alexays/Waybar#3430]
hi, im using wayland with archlinux,
when starting wayland everything works fine but if i will change the config file of waybar (adding module for instance) and then restart waybar i will get:
```
==> ERROR: Cannot fetch updates
[2024-07-10 13:49:46.256] [error] Host: GDBus.Error:org.freedesktop.DBus.Error.UnknownObject: No such object path '/StatusNotifierWatcher'
[2024-07-10 13:49:46.357] [error] Host: GDBus.Error:org.freedesktop.DBus.Error.UnknownObject: No such object path '/StatusNotifierWatcher'
[2024-07-10 13:49:46.370] [warning] Requested height: 30 is less than the minimum height: 35 required by the modules
[2024-07-10 13:49:46.370] [info] Bar configured (width: 1920, height: 35) for output: HDMI-A-1
[2024-07-10 13:49:46.370] [warning] Requested height: 30 is less than the minimum height: 35 required by the modules
[2024-07-10 13:49:46.370] [info] Bar configured (width: 1920, height: 35) for output: eDP-1
```

waybar start with the following script
```
#!/bin/sh

killall waybar

waybar -c ~/dotfiles/waybar/config.jsonc & -s ~/dotfiles/waybar/style.css
```
but even if i will manualy just put in the terminal `waybar &` i will get this error, i also tried killing the `nm-applet` process and the restarting waybar but it didnt work. 
thank you very much

update: i noticed that it happens when changing the css file

**Comment by RobertMueller2** [comment]
Can you try if the PR mentioned above fixed the problem? It's now in git and included with 0.10.4.

**Comment by Shmerlard** [comment]
yup looks like it was solved, thanks!
